### DISCUSSION TITLE
Feature request: Display users logged

### DISCUSSION THREAD

**Discussion by morenod** [nicolargo/glances#3234] [General]
Before filling this issue, please read the manual (https://glances.readthedocs.io/en/latest/) and search if the bug do not already exists in the database (https://github.com/nicolargo/glances/issues).

#### Description

For integration with activity sensors like home-assistant, information about users logged on the system, displaying if the session is active or locked.

Sending events to the log on login/logout/lock/unlock will be also useful

#### Versions

* Glances & psutil (glances -V):
* Operating System (lsb_release -a):

#### Logs

You can also pastebin the Glances logs file (https://glances.readthedocs.io/en/latest/config.html#logging)

**Comment by nicolargo** [comment]
You can use AMP to do this kind of thinks: https://github.com/nicolargo/glances/blob/develop/docs/aoa/amps.rst

In the glances.conf file, add:

```
[amp_loggeduser]
enable=false
refresh=30
one_line=false
command=w
```
